src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public interface Figure {
+    double getArea();
+
+    void draw();
+
+    Color getColor();
```

Comment:
in order to implement Interface Segretation Principle, divide this interface into `AreaCalculator` and `Drawable`.
extracting getters/setters into interfaces is redundant, so just make `getColor` as regular method in figures
as result, this class should be deleted